File: rocket_back/rocket/src/main/java/com/uber/rocket/dto/RideHistory.java
package com.uber.rocket.dto;

import lombok.*;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class RideHistory {
    private String driver;
    private double price;
    private double duration;
    private String start;
    private String end;

    private String date;

    List<DestinationDTO> destinations;
    String routeLocation;
    private Long id;
}

File: rocket_front/rocket/src/app/interfaces/RideHistory.ts
import { Destination } from "./Destination";

export interface RideHistory {
  'start': string,
  'end': string,
  'driver': string,
  'duration': string,
  'price': string,
  'id': string,
  'date':string,
  rideLocation: string,
  destinations: Destination[];
}
